zathura/utils.c

```diff
@@ -125,6 +128,82 @@ static gboolean search_current_index(GtkTreeModel* model, GtkTreePath* UNUSED(pa
   return FALSE;
 }
 
+static bool find_substring(const char* source_str, const char* search_str) {
```

Comment:
Find the `search_str` in `source_str`, as opposed to only match from the beginning of `source_str`